python-avd/pyavd/_eos_designs/schema/schema_fragments/underlay_ospf_authentication_schema.yml

```
@@ -0,0 +1,40 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  underlay_ospf_authentication:
+    documentation_options:
+      table: ospf-settings
+    type: dict
+    keys:
+      enabled:
+        type: bool
+        default: false
+      message_digest_keys:
+        type: list
```

Should we add some min length and max length on this list? As far as I understand the doc, EOS supports one or two keys.
```suggestion
        type: list
        required: true
        primary_key: id
        min_length: 1
        max_length: 2
```